src/APIFunctions/Auth.js

```diff
@@ -93,7 +92,11 @@ export async function checkIfUserIsSignedIn() {
   }
 
   await axios
-    .post(GENERAL_API_URL + '/Auth/verify', { token })
+    .post(GENERAL_API_URL + '/Auth/verify', {}, {
+      headers: {
+        Authorization: `${token}`
```

Comment:
make the header look like `Bearer <token>`, this follows a convention
https://stackoverflow.com/a/33281233